run/opencl/pdf_kernel.cl

```diff
@@ -484,7 +484,7 @@ void pdf_r5(__global const uchar *pwbuf,
 #endif
 
 		uint gidx = gid * NUM_INT_KEYS + mi;
-		uint W[(PLAINTEXT_LENGTH + 8 + 63 + 9) / 64 * 16]; // Max. 133 bytes, three limbs
+		uint W[(PLAINTEXT_LENGTH + 8 + 63 + 9) / 64 * 16];
```

Comment:
I see now this instance of PLAINTEXT_LENGTH can be 127 due to that truncation. Doesn't matter much but I'll fix it and force push.

Reply:
Oh, and I need to protect those shorter buffers from overrun by the device side mask as well.

Reply:
Done and force pushed. Sadly, decreasing to 127 here made the test failure reappear (but with R5 hashes, which makes sense as the change was in that kernel).